sparkler-core/bin/dockler.dev.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
```

Comment:
This script seems fine... but I feel that we should update [the documentation](http://irds.usc.edu/sparkler/dev/development-environment-setup.html) if this is being proposed as the official way to establish a development environment.

Reply:
I am not sure how to access that documentation and if it is still current. I added documentation to the official wiki of the project. https://github.com/USCDataScience/sparkler/wiki/Elasticsearch-&-Dev-Container
Please reopen if this is still an issue.